docs/pre-release/releaseNotes.yml

```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
```

Comment:
We should link to the documentation for how to configure the logs for this one and include an image how what the improve IP log statement looks like.